cmd/osm-controller/validate.go

```diff
@@ -32,18 +30,9 @@ func validateCLIParams() error {
 }
 
 func validateCertificateManagerOptions() error {
```

Comment:
Why is this only called in the osm-controller?

Reply:
removed, it's an unnecessary additional validation